FantasmoSDK/app/build.gradle

```diff
@@ -49,26 +51,35 @@ android {
 
     flavorDimensions "env"
     productFlavors {
+        buildTypes{
+            release{
```

Comment:
The build types might be more aptly named `production` and `development`?

What do you think about simplifying the build types entirely and just have a single build config field, `FM_API_BASE_URL`, which is just the host (i.e. `api.fantasmo.io`), and then having the methods add the resource and version parts, something like:

```
fun getIsLocalizationAvailableURL(): String {
    return BuildConfig.FM_API_BASE_URL + "/v1/isLocalizationAvailable"
}
```

Reply:
I agree on the build types simplification, that's actually a good idea. Not sure what you meant by naming `production` and `development`.

Reply:
Probably no need to change the names, I think associating `release` with production env and `debug` with development env makes sense anyhow :)